### ISSUE TITLE
sass npm linked packages compilation times too slow

### ISSUE THREAD

**Issue by PayteR** [nuxt-community/style-resources-module#44]
Hi, I came across an issue, when I @import *.scss files from liked node packages. https://docs.npmjs.com/cli/link.html 

I use a global main.scss file, that includes Bulma packages by cascading imports. That means I load 
```
main.scss 
> _all.scss 
>> components/_all.scss
>>> navbar.scss
>>>> ~bulma_scss/scss/components/navbar.scss 
```
I import all Bulma's scss files like this and there are some mine packages too so it's about 200+ files. Compilation time in dev mode is about 10s, it's slow, but still, I can live with that. The issue is when I import any scss file from my package, that is in node_modules linked by npm's link https://docs.npmjs.com/cli/link.html then compilation time is more than 60s and that's too much. 

Is there any way to solve this issue or any idea? I don't have this issue in my other non-nuxt project's that compiles these files with webpack's sass-loader. Thank you.

**Comment by manniL** [comment]
See #35, #42 #39. Please ping if that didn't help.

**Comment by PayteR** [comment]
@manniL I made several tests, and I came to the conclusion that when @imports are in template files, then my approach is not slow and this issue is not present. So this issue is fixed by moving @imports into template files.

I knew that I should load by '@nuxtjs/style-resources' just variables and mixins, but I want to have global styles that are included anywhere. Now I have this in my layouts/default.vue 

![image](https://user-images.githubusercontent.com/1248565/56085593-4d5a5c80-5e46-11e9-9a5b-fbdb0da89032.png)

This is not an elegant solution IMHO, is there any recommended approach to load global styles into the nuxt app? Thank you

**Comment by manniL** [comment]
If you want global styles, please use `css` in your `nuxt.config.js`.

As stated in the linked issues, all the files linked in the style-resources config are injected into **every** component and increasing the CSS drastically if you don't just add variables and co.

**Comment by PayteR** [comment]
Thank you, I moved that imports into css: [] array. I made a several tests and result is that there is no significant speed difference. 

I added this to config too 
```
  build: {
    parallel: true, // don't know, if it works, maybe
    cache: true, // don't know, if it works, maybe
    hardSource: true, // don't know, if it works, maybe
}
```
Now when scss is cached, compilation time is < 1s, which is great.  When i change variable in styleResources, then compilation time is about 3s and thats sufficient. 

About my original issue with linked npm package - there is no change. There is still performance issue that cause compilation time > 20s when even one npm package is symlinked in node_modules. My solution will be just not to do that. I will sync every my change into packages manualy. I already spend a lot of time on this, so if there is no simple solution, then leave it.
